### PR TITLE
TINY-8555: Added changelog entries for TinyMCE 6.0.0 and 6.0.1

### PR DESCRIPTION
**Pull request by lnewson**
Related Ticket: TINY-8555

Description of Changes:
* Added the changelog entries for 6.0.0 and 6.0.1

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

**Review by tiny-ben-tran — APPROVED** [review]

**Review by metricjs — APPROVED** [review]

File: modules/ROOT/pages/changelog.adoc [hunk 1]
```diff
@@ -5,4 +5,179 @@
...
+=== Fixed
+* The object returned from the `editor.fire()` API was incorrect if the editor had been removed.
+* The `editor.selection.getContent()` API did not respect the `no_events` argument.
+* The `editor.annotator.remove` API did not keep selection when removing the annotation.
+* The `GetContent` event was not fired when getting `tree` or `text` formats using the `editor.selection.getContent()` API.
+* The `beforeinput` and `input` events would sometimes not fire as expected when deleting content.
+* The `table` plugin would sometimes not correctly handle headers in the `tfoot` section.
+* The `silver` theme UI was incorrectly rendered before plugins had initialized.
+* The aria labels for the color picker dialog were not translated.
+* Fixed sub-menu items not read by screen readers. Patch contributed by westonkd.
```
Comment (EkimChau) [inline modules/ROOT/pages/changelog.adoc:125]: duplication with Line 129

**Review by EkimChau — APPROVED** [review]
Remove 1 duplication on Line 129

File: modules/ROOT/pages/changelog.adoc (hunk 1 above)
Comment (lnewson) [reply modules/ROOT/pages/changelog.adoc:125]: This was copied from the original, so looks like we have the same issue there. I suspect this was caused by the reordering done pre 6.0 😢

Comment (lnewson) [reply modules/ROOT/pages/changelog.adoc:125]: Have removed the duplicate line here but we'll also need to do the same in the tinymce repo.

**Pull request merged**